utility/gov.go

```diff
@@ -559,46 +559,46 @@ func (u *UtilityContext) getBigIntParam(paramName string) (*big.Int, typesUtil.E
 	if er != nil {
 		return nil, er
 	}
-	value, err := store.GetStringParam(paramName, height)
+	value, err := store.GetParameter(paramName, height)
 	if err != nil {
 		log.Printf("err: %v\n", err)
 		return nil, typesUtil.ErrGetParam(paramName, err)
 	}
-	return typesUtil.StringToBigInt(value)
+	return typesUtil.StringToBigInt(value.(string))
 }
 
 func (u *UtilityContext) getIntParam(paramName string) (int, typesUtil.Error) {
 	store, height, er := u.GetStoreAndHeight()
 	if er != nil {
 		return 0, er
 	}
-	value, err := store.GetIntParam(paramName, height)
+	value, err := store.GetParameter(paramName, height)
 	if err != nil {
 		return typesUtil.ZeroInt, typesUtil.ErrGetParam(paramName, err)
 	}
-	return value, nil
+	return value.(int), nil
 }
 
 func (u *UtilityContext) getInt64Param(paramName string) (int64, typesUtil.Error) {
```

Comment:
In these helpers `getInt64Params`, etc. We should validate if the type casting you're doing (e.g. `.(int)`) is valid and return an error if `!ok`.

Ref: https://stackoverflow.com/a/11580545/768439